web/context_test.go

```diff
@@ -13,6 +14,7 @@ import (
 type myHandler struct{}
 
 func (m myHandler) ServeHTTP(e *RequestEvent, w http.ResponseWriter, r *http.Request) {
+	time.Sleep(10 * time.Millisecond)
```

Comment:
Why is this needed?

Reply:
Otherwise the response happens so fast that the latency gets rounded down to `0`, which fails the test, which is looking for `0.`. I could also change the test if you prefer.

Reply:
Ah that makes sense, can you just add a comment explaining that.